.github/workflows/link_check.yml

```
@@ -0,0 +1,85 @@
+name: Check Docs & Links
+
+on:
+  pull_request:
+    branches:
+      - main
+  push:
+    branches:
+      - main
+  schedule:
+    - cron: "0 5 * * *"
+  workflow_dispatch:
```

From the code below, it looks like you only want `.ipynb` files, in that case do this
```suggestion
on:
  push:
    branches: ["main"]
  pull_request:
    paths:
      - '**.ipynb'
  workflow_dispatch:
```
and no need for scheduling, just have it run in PRs and on commits to main

---

Ah, unless this is because we want to have a recurring check. Prob makes sense. In that case ya keep the scheduling.